schedule/microos/suse_microos_dvd.yaml

```diff
@@ -2,7 +2,15 @@ name:           suse_microos
 description:    >
     Maintainer: jalausuch@suse.com, qa-c@suse.de.
     SUSE MicroOS tests
+conditional_schedule:
+    bootloader:
+        ARCH:
+            'aarch64':
+                - installation/bootloader_uefi
+            'x86_64':
+                - installation/bootloader
 schedule:
+    - '{{bootloader}}'
     - installation/bootloader
```

Comment:
Maybe it is a bit hackish, but we can use `installation/bootloader_uefi` for all 3 cases. EFI boot, MBR and for aarch64.

https://openqa.opensuse.org/tests/1421879#

Reply:
I will remove that from this PR as it's a different topic and would require more time for VRs.